pkg/policies/policies.go

```diff
@@ -16,13 +16,179 @@
 package policies
 
 import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"path/filepath"
+
 	v1 "github.com/chainloop-dev/chainloop/app/controlplane/api/workflowcontract/v1"
+	"github.com/chainloop-dev/chainloop/internal/attestation/crafter"
+	v12 "github.com/chainloop-dev/chainloop/internal/attestation/crafter/api/attestation/v1"
+	"github.com/chainloop-dev/chainloop/internal/casclient"
 	"github.com/chainloop-dev/chainloop/pkg/policies/engine"
 	"github.com/chainloop-dev/chainloop/pkg/policies/engine/rego"
+	"github.com/sigstore/cosign/v2/pkg/blob"
+	"google.golang.org/protobuf/encoding/protojson"
```

Comment:
This is super nitpick sorry...could you organize the imports like:
```
standard library

chainloop and current source code

third party dependencies
```

Reply:
It's exactly that way. Otherwise the linter would complain (I also have goimports configured locally)

Reply:
well, it seems goimports is not configured properly, checking ...

Reply:
Done!